From 99d7f3a7f56cc8839b9e2774f13bfca538a36035 Mon Sep 17 00:00:00 2001
From: Jakub Ruzicka <jruzicka@redhat.com>
Date: Mon, 9 Dec 2013 15:58:31 +0100
Subject: [PATCH] Remove runtime dependency on python-pbr

---
 heatclient/__init__.py | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/heatclient/__init__.py b/heatclient/__init__.py
index e589bb4..25b68fa 100644
--- a/heatclient/__init__.py
+++ b/heatclient/__init__.py
@@ -10,7 +10,4 @@
 #   License for the specific language governing permissions and limitations
 #   under the License.
 
-import pbr.version
-
-
-__version__ = pbr.version.VersionInfo('python-heatclient').version_string()
+__version__ = "REDHATHEATCLIENTVERSION"
